ArgumentValidator.php

```diff
@@ -153,4 +165,26 @@ private function validateClass($expectedClassName, $actualClassName)
             ));
         }
     }
+
+    /**
+     * Checks if the installed ExpressionLanguage component version is supported
+     * by the installed version of the DependencyInjection component.
+     *
+     * Starting with Symfony 2.6, the DependencyInjection component requires the
+     * ExpressionLanguage component to be of version 2.6 or higher.
+     *
+     * @return bool
+     */
+    private function isExpressionLanguageSupported()
+    {
+        if (!method_exists('Symfony\Component\DependencyInjection\Definition', 'getFactory')) {
+            return true;
+        }
+
+        if (interface_exists('Symfony\Component\ExpressionLanguage\ExpressionFunctionProviderInterface')) {
```

Comment:
@xabbuh Also, I think that the check should not be for the interface, but for `Symfony\Component\ExpressionLanguage\ExpressionLanguage`, since that would seem like a stable class, which is always available, right? The rest should be covered by proper dependency versions based on `composer.json`.